eng/SharedFramework.External.props

```diff
@@ -7,6 +7,8 @@
 <Project>
 
   <!-- For the targeting pack, always use packages with PatchVersion=0 -->
+  <!-- !!! Need to get ref/ assemblies for these packages some other way in order to feed correct information into Roslyn. !!! -->
```

Comment:
Why the exclamation points - was this a comment to yourself that you removed later?

Reply:
Yes, I removed this later.